Assets/HoloToolkit/Sharing/Tests/ImportExportAnchorManager.cs

```diff
@@ -116,18 +116,19 @@ public bool AnchorEstablished
     /// </summary>
     RoomManagerAdapter roomManagerCallbacks;
 
-    void Start()
+    void Awake()
```

Comment:
Aren't scene objects loaded before the first Awake call is made?
If yes, Singleton<T> already accounts for that by calling FindObjectOfType<T> if the instance doesn't exist.